src/Microsoft.VisualStudio.Editors/AttributeEditor/PermissionSetService.vb

```diff
@@ -113,12 +114,12 @@ Namespace Microsoft.VisualStudio.Editors.VBAttributeEditor
                     identityList = StringToIdentityList(strExcludedPermissions)
                 End If
 
-                Return SecurityUtilities.ComputeZonePermissionSet( _
-                    strTargetZone, _
-                    projectPermissionSet, _
+                Return SecurityUtilities.ComputeZonePermissionSet(
+                    strTargetZone,
+                    projectPermissionSet,
                     identityList)
 
-            Catch ex As Exception
+            Catch ex As Exception When Common.Utils.ReportWithoutCrash(ex, NameOf(ComputeZonePermissionSet), NameOf(PermissionSetService), considerExceptionAsRecoverable:=True)
```

Comment:
Example for Category 3 (Silently swallowing System.Exception without re-throwing)